crates/database/src/orders.rs

```diff
@@ -264,7 +264,17 @@ pub struct Quote {
     pub buy_amount: BigDecimal,
 }
 
+pub async fn insert_quotes(ex: &mut PgConnection, quotes: &[Quote]) -> Result<(), sqlx::Error> {
+    for quote in quotes {
+        insert_quote(ex, quote).await?;
+    }
+    Ok(())
+}
+
 pub async fn insert_quote(ex: &mut PgConnection, quote: &Quote) -> Result<(), sqlx::Error> {
+    /// For ethflow orders, due to reorgs, different orders
+    /// might be inserted with the same uid. Hence, we need
+    /// to update quote entries in the database on conflicts
```

Comment:
We could split the query up into one that allows conflicts and one that doesn't. Or it could be a parameter to the function "override on conflict". That's slightly nicer for non ethflow orders because there it really would be an error. Not sure if needed though.